src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -131,6 +134,8 @@ public Object instanceMainPostParseArgs() {
         this.defaultHeaders.add(new StringHeader("Started on: " +
                                 startDateTime.format(DateTimeFormatter.ofLocalizedDateTime(FormatStyle.LONG))));
 
+        LoggingUtils.sendLogToFile(logToFile);
+
         LoggingUtils.setLoggingLevel(VERBOSITY);  // propagate the VERBOSITY level to logging frameworks
 
         for (final File f : TMP_DIR) {
```

Comment:
So I think we want to keep that message going to stderr, because we want it output always even with restrictive logging modes.  We want to make sure it also is duplicated in the log file though.